applications/settings/desktop_settings/desktop_settings_custom_event.h

```diff
@@ -0,0 +1,26 @@
+#pragma once
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+typedef enum {
+    // reserve 100 for button presses, submenu selections, etc.
```

Comment:
input served through another message queue, where this comment came from?

Reply:
Some other apps directly send events received from views. Because of that they need to reserve enum values at the beginning.